pom.xml

```
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.20</version>
+    <version>4.19</version>
```

Probably need to merge with the master branch to not have this revert to an older version
```suggestion
    <version>4.20</version>
```

---

I have mention about reverting of version from 20 to 19 in the gitter chat and the issue that I am facing when the version is set to 20. Also I have mentioned some cases over which the binding is failing in the chat as well, it will certainly make you more informed when performing interactive testing.